src/app.py

```diff
@@ -47,7 +57,41 @@ def _log_filename() -> str:
 setup_logger(log_file=webhook_logs_dir / _log_filename())
 
 
+def signature_validator(func: Callable[[], tuple[str, int]]) -> Callable[[], tuple[str, int]]:
```

Comment:
should this be defined in the validation module?

Reply:
I put it here to avoid having flask request specific code in the validation module. I removed the decorator (probably overkill anyway, as it is only used in one endpoint).